sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/SpecializedBlobClientBuilder.java

```diff
@@ -502,4 +507,11 @@ public SpecializedBlobClientBuilder serviceVersion(BlobServiceVersion version) {
         this.version = version;
         return this;
     }
+
+    private BlobServiceVersion fromClientVersion(String version) {
```

Comment:
this can be private static because it doesn't depend on any member variables from what I can see.

Reply:
Done